tests/acceptance/features/api/api_version.feature

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+Feature: Get information about GlanceSync API
+  As a GlanceSync API user
+  I want to see some information of the API that is running
+  So that I can check the version, owner, documentation location and other info parameters.
+
+  GlanceSync API operations:
+    GET /
+
+
+  Scenario: Request API information when user is authenticate.
+    Given the API running properly
+    And   the user is successfully authenticate
```

Comment:
Yes, I know.There are other scenarios that test the same flow but without authentication.